APMrover2/commands_logic.cpp

```diff
@@ -228,9 +225,17 @@ void Rover::do_nav_wp(const AP_Mission::Mission_Command& cmd, bool stay_active_a
     // this is the delay, stored in seconds
     loiter_duration = cmd.p1;
 
+    // get heading to following waypoint (auto mode reduces speed to allowing corning without large overshoot)
+    // in case of non-zero loiter duration, we provide heading-unknown to signal we should stop at the point
+    float next_leg_bearing_cd = MODE_NEXT_HEADING_UNKNOWN;
```

Comment:
I was thinking we would convert it all to degrees in a follow up PR.  I made the new argument in centi-degrees because the mode class already has a _yaw_error_cd member and I didn't want to have it half-and-half nor did I want to increase the scope of the changes anymore.